app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -15,11 +15,24 @@
       </span>
     {{/if}}
   </div>
-  <div class="column">
-    <div class="field">
-      {{input type='number' name='ticket_quantity' placeholder=(t 'Quantity') value=ticket.quantity min=1}}
+  {{#if (eq ticket.type 'donation')}}
+    <div class="column two wide">
```

Comment:
@uds5501 @shreyanshdwivedi Could you please suggest a way to align min. quantity, max.quantity with options?